README.md

```diff
@@ -1 +1,97 @@
-# microverse-linter-check
\ No newline at end of file
+# HTML & CSS3 Course
```

Comment:
- [ ] Kindly use the Microverse template for the README file instead of your README file you can find it [here](https://github.com/microverseinc/readme-template). and replace the info with your info in order to make your README professional as it's the first thing the viewer sees when he visits the repo.